core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -23,10 +23,16 @@ import scala.annotation.unchecked.uncheckedVariance
 import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 
-sealed abstract class IO[+A] private () extends IOPlatform[A] {
+sealed abstract class IO[+A] private () {
```

Comment:
Actually I have some changes locally which need this to remain: `fromCompletableFuture` (JVM-only) and `fromPromise` (JS-only). So in other words, `IORuntime` is just kind of a separate thing entirely.

Reply:
OK will restore as empty files.